SQLClient/src/internal/queryMaker/queryMaker.go

```diff
@@ -0,0 +1,100 @@
+package queryMaker
+
+import (
+	"errors"
+	"fmt"
+	"github.com/ECE356-Final-Project/SQLClient/src/internal/dbConnector"
+	"github.com/ECE356-Final-Project/SQLClient/src/internal/utils"
+	_ "github.com/go-sql-driver/mysql"
+	sql "github.com/jmoiron/sqlx"
+	"log"
+	"strconv"
+	"strings"
+)
+
+// QueryMaker struct{Db dbInterface, sql }
+type QueryMaker struct {
+	Db dbConnector.DBConnector
+}
+
+func (q *QueryMaker) connectToDB() (dbConnector.DBConnector, error) {
+	//fmt.Println("------------ 2020 Elections Sentiment Analysis Results Database ------------")
+	//fmt.Println("Connecting to DB...")
+
+	connString := "root:root@tcp(127.0.0.1:3307)/Election"
+	return sql.Open("mysql", connString)
+}
+
+func (q *QueryMaker) doQuery(input string) (*sql.Rows, error) {
+	results, err := q.Db.Queryx(input)
+	if err != nil {
+		log.Printf("error doing query %q: %s", input, err.Error())
+		return nil, err
+	}
+	return results, nil
+}
+
+func NewQueryMaker(dbs ...dbConnector.DBConnector) *QueryMaker {
+	qm := &QueryMaker{}
+
+	if len(dbs) > 0 { // use optional dbConnector for testing
+		qm.Db = dbs[0]
+	} else {
+		db, err := qm.connectToDB()
+		if err != nil {
+			log.Fatalf("Could not connect to DB: %s", err.Error())
+		}
+		qm.Db = db
+	}
+	return qm
+}
+
+func getStringForGetVotesForCandidate(candidate, county string, states []string) string {
```

Comment:
I think having them as one file is fine since its the same struct. We may consider moving string construction to another package if needed but we can refactor that pretty easily since they are already in seperate functions